staging-service/src/main/java/com/github/cafdataprocessing/services/staging/dao/filesystem/SubBatchWriter.java

```diff
@@ -63,14 +61,7 @@ private void createSubBatchOutStream() throws StagingException
     {
         //Make a new subbatch file
         final String subBatchFileName = BatchNameProvider.getSubBatchName();
-        final Path inProgressPath = Paths.get(inProgressBatchFolder.toString());
-        final Path subBatchPath = inProgressPath.resolve(subBatchFileName);
-
-        if (!subBatchPath.normalize().startsWith(inProgressPath)) {
```

Comment:
Isn't the subbatch filename completed generated by us?  If so then we shouldn't have to check this as there is no external input to it.